biogrid/libs/biogrid-simulator/src/lib/biogrid/biogrid-class.ts

```diff
@@ -1,3 +1,21 @@
+/**
+ * @file defines the @class Biogrid which describes the microgrid.
+ * This class @implements @interface Grid which add the functionalities microgrid
+ * The microgrid defines the following components:
+ * 1. @param smallBatteries which is the small batteries storing power in the grid
+ * 2. @param largeBatteries which is the large batteries storing power in the grid
+ * 3. @param solarPanels which is the solar panels in supplying power in the grid
+ * 4. @param state which defines the state graph for the grid items
```

Comment:
remove params? these could change in the future, and the naming of the params is pretty self explanatory imo